README.md

```diff
@@ -1,2 +1,35 @@
-# gonome
+# Gonome
 Genome Edit Design Service
+
+## Goal
+Build a software service for designing genomic edit implementations across therapeutically relevant reference genomes and custom genomes.
+
+### Feature Requirements
```

Comment:
It would be good to call out some expectations around unit testing, even setting some coverage threshold that we want to maintain. I'd saying something in the 70%-80% feels safe and reasonable, but I'm just making those numbers up on the spot. Not sure if this is a "feature requirement" per se, but it should live somewhere in this doc.

Reply:
Good point, added a Development Requirements section.